### ISSUE TITLE
Shared RES Tags Feature Request

### ISSUE THREAD

**Issue by DaGeek247** [honestbleeps/Reddit-Enhancement-Suite#827]
I have a features request, maybe for the paid version of RES. The idea is shared RES tags. Every one who opts in to the feature has their tag stored on a database, and everyone can change the tags. I think it would interesting to see what happened, and which famous redditor got what tags. It would also be interesting to see what or if someone else tagged you, especially if tagging yourself wasn't allowed.

It may need to be in the paid version because website database, especially ones like this would need to be accessed a lot in a short amount of time. If it became too much maybe we could set limits on how many times someone could tag other people publicly. Or perhaps a user wide push where the tags are updated daily instead of instantly.

To sum up the idea, shared RES tags hosted on a database, available to everyone.

**Comment by erikdesjardins** [comment]
Closing as RES Pro has been dead for a while now
